app/services/amr/perse_upsert.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module Amr
+  module PerseUpsert
+    def self.perform(meter, reload: false)
+      config = AmrDataFeedConfig.find_by!(identifier: 'perse-half-hourly-api')
+      date = 14.months.ago
+      date = latest_reading_date(config, meter) || date unless reload
```

Comment:
Is it possible to add a test to check that the right date is used if reload is / isn't provided?

Reply:
added to https://github.com/Energy-Sparks/energy-sparks/blob/af6a502a7f0ef04c43b97da5c4433d5dabc02934/spec/system/meter_management_spec.rb#L302 to check the reload button doesn't use the last reading date - does that seem ok?

Reply:
Yes. In an ideal world you'd probably want to add a test to perse_upsert_spec.rb but ok if you're happy it's been done here.